change/@azure-msal-common-e77b173d-7e21-4295-82f2-5dcc3edf653f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Let's make this minor since we are adding APIs to the surface of the logout request